neurom/apps/cli.py

```diff
@@ -42,3 +44,46 @@ def view(input_file, plane, backend, realistic_diameters):
 
     if backend == 'matplotlib':
         plt.show()
+
+
+@cli.command(short_help='Morphology statistics extractor, more details at'
+                        'https://neurom.readthedocs.io/en/latest/morph_stats.html')
+@click.argument('datapath', required=False)
+@click.option('-C', '--config', type=click.Path(exists=True, dir_okay=False),
```

Comment:
Is there a reason why you are using a capital letter for the `-C` ?
I see the `-I` also (capital i) for the ignored-exceptions.

Reply:
backward compatibility only